core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -110,6 +110,7 @@
 
   skills-list .delete-icon {
     padding-left: 0.8vw;
+    cursor: pointer;
```

Comment:
Add this in alphabetically sorted order.

Reply:
Done